solutions/gold/ac-smallmultiple.mdx

```diff
@@ -90,5 +89,40 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+from collections import deque
+
+INF = float("inf")
```

Comment:
no need for the constant now so just replace it

Reply:
done